Update chat.md with commteam chatroom

Echolon opened the pull request.

cal0pteryx commented on content/community/chat.md:
```diff
@@ -9,26 +9,29 @@ aliases:
 
 The XSF hosts a variety of chat rooms and all archives are available online. Participation in the XSF's discussion channels is subject to the [Community Code of Conduct](https://xmpp.org/extensions/xep-0458.html).
 
-### Archives
```
This should stay ###, since it's about archives. The points following this chapter contain archive links, but are not exclusively about archives. They contain links to live chats.

Echolon replied:
reverted e571b03

cal0pteryx commented on content/community/chat.md:
```diff
@@ -9,26 +9,29 @@ aliases:
+## Archives
 
-Logs of XSF public chatrooms are available at [logs.xmpp.org](https://logs.xmpp.org/).
+All logs of public XSF chatrooms are available at [logs.xmpp.org](https://logs.xmpp.org/).
```
What's the intention behind this change?

Echolon replied:
I think my formulation is more precise

cal0pteryx commented:
I'm not sure why you changed the order? MattJ reordered things in 297187456d8187692e5fabfdbd97340b6cd8301b recently. Can you explain why these changes are necessary?

Echolon commented:
> I'm not sure why you changed the order? MattJ reordered things in [2971874](https://github.com/xsf/xmpp.org/commit/297187456d8187692e5fabfdbd97340b6cd8301b) recently. Can you explain why these changes are necessary?

I ordered against what the XSF is here for + I added the Communication team

Echolon commented:
> > I'm not sure why you changed the order? MattJ reordered things in [2971874](https://github.com/xsf/xmpp.org/commit/297187456d8187692e5fabfdbd97340b6cd8301b) recently. Can you explain why these changes are necessary?
> 
> I ordered against what the XSF is here for + I added the Communication team

reverted 3b5a97f
